.github/workflows/ci.yml

```diff
@@ -747,7 +852,7 @@ jobs:
   notify_on_failure:
     needs:
       [
-        ci-is-on-main-repo,
+        check-actor-permissions,
```

Comment:
Should you add `check-actor-permissions` to `notify_on_release` also?

Reply:
`build-n-publish` needs it so it would skip, but it's better to add it to every step explicitly. I'll update the PR.